v23.1/fips-compliance.md

```diff
@@ -0,0 +1,355 @@
+                        {% endif %}
+                    {% endif %}
+        </td>
+                {% endif %}
+
+                {% comment %}Docker image{% endcomment %}
+        <td>
+            <code>cockroachdb/cockroach:{{ r.release_name }}-fips</code>
+        </td>
+    </tr>
+                {% endif %}
+            {% endfor %} {% comment %}Releases {% endcomment %}
+</table>
+        {% endif %}
+    {% endfor %}{% comment %}Sections {%endcomment %}
+{% else %}
+No FIPS-ready runtimes are available at this time. Please check again later.
+{% endif %}
+
+<a id="install"></a>
+
+## Install the FIPS-ready CockroachDB runtime
+
+After you [download](#downloads) a FIPS-ready CockroachDB binary, install it in the same way as the standard binary. Refer to [Install CockroachDB on Linux](install-cockroachdb-linux.html).
+
+{{site.data.alerts.callout_info}}
+Upgrading an existing CockroachDB cluster's binary in-place to be FIPS-ready is not supported.
+{{site.data.alerts.end}}
+
+### Verify that CockroachDB is FIPS-ready
+
+If the CockroachDB binary is FIPS-ready, the string `fips` is appended to the Go version in the `cockroach version` command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach version |grep fips
+~~~
+
+~~~ shell
+go1.19.5fips
+~~~
+
+This indicates that CockroachDB was built using [Red Hat's Go FIPS with OpenSSL toolchain](https://github.com/golang-fips/go).
+
+## Use the FIPS-ready CockroachDB Docker image
+
+1. If necessary, enable FIPS mode on the Docker host. The FIPS-ready CockroachDB Docker image must run on a Docker host with FIPS mode enabled. To enable FIPS mode in the Docker host kernel, refer to [Enable FIPS mode](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening) in Red Hat's documentation. To verify that FIPS mode is enabled, refer to [Verify that the kernel enforces FIPS mode](#verify-that-the-kernel-enforces-fips-mode).
+1. Visit [Releases](/docs/releases/index.html?filters=docker) and copy the name of a FIPS-ready Docker image tag.
+1. Pull the Docker image locally, create a new container that uses it, run the container, and attach to it. This example gives the running container the name `cockroachdb-fips-container`. Replace `<image_tag>` with the Docker image tag you copied.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker run <image_tag> --name="cockroachdb-fips-container" -i
+    ~~~
+
+1. In the Docker host, [verify that the kernel enforces FIPS mode](#verify-that-the-kernel-enforces-fips-mode).
+
+    {{site.data.alerts.callout_info}}
+    Do not attempt to run the `sysctl` test from within the running container.
```

Comment:
Is this correct?

```suggestion
    Do not attempt to run the `sysctl` test from within the running container. Run the command from the Docker host rather than from the Docker container.
```

Reply:
Yes, Docker containers can't interact with the Docker host's `systemctl`. It's a separation of concerns. Thanks for the suggestion.